src/dependabot/verified_commits.ts

```diff
@@ -21,7 +21,7 @@ export async function getMessage (client: InstanceType<typeof GitHub>, context:
 
   // Don't bother hitting the API if the PR author isn't Dependabot
```

Comment:
I think this is the reason the check exists within the code... the concern is that users may not realize they need to set that, and so may fire off spurious requests... so it's defensive coding to protect the API from ignorant/careless users. 

That said, I agree the volume of calls here is probably pretty low, so it may be fine to remove this... 🤷‍♂️

Reply:
I filed https://github.com/dependabot/fetch-metadata/issues/332 to track this discussion as it's out of scope of this PR.